package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "react-gojs",
+  "version": "1.0.0",
+  "description": "GoJS React integration",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "node scripts/clean.js && tsc",
+    "watch": "node scripts/clean.js && tsc -w",
+    "test": "node scripts/test.js --env=jsdom",
+    "lint": "tslint -c tslint.json 'src/**/*.{ts,tsx}'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/xcomponent/react-gojs.git"
+  },
+  "keywords": [
+    "gojs",
+    "react"
+  ],
+  "author": "XComponent <dev@xcomponent.com> (http://www.xcomponent.com)",
+  "license": "MIT",
```

Comment:
Some projects are Apache 2... what's the rule on those things?

Reply:
Just a mistake (on my personal stuff I use MIT). I will change it